topics/transcriptomics/tutorials/scrna-umis/tutorial.md

```diff
@@ -200,13 +200,13 @@ This then provides us with the true count of the number of true transcripts for
 > >
 > >   e.g.
 > >
-> >  | BC:Cell | BC:UMI | Maps to Gene |
+> >  | BC:Cell | Maps to Gene | BC:UMI | Maps to Gene |
```

Comment:
@nomadscientist should the last column header also be removed?

Reply:
Good point, yes!